code-style/scss/README.md

```diff
@@ -1,52 +1,257 @@
+
+---
+
+## BEM
+
+BEM stands for block element modifier, which is the order you follow for naming with BEM.
+
+#### Block
+
+`.block` is the parent wrapper of the component. This would be something like site navigation, the hero section, or a footer. Any container that holds other code could be a block.
+
+#### Element
+
+`.block__element` is an element inside the component, or inside the parent wrapper. This could be a title or a link. It's anything inside the parent wrapper. It's important here to note that BEM isn't meant to reflect nesting. For example, say you have a section that includes an image element inside an anchor element:
+
+```html
+<section>
+  <a href="#">
+    <img src="#">
+  </a>
+</section>
 ```
 
-**Subcomponents** are bits and pieces of a module that are specific to that module. A module is separated from a subcomponent with *double--hyphens*:
-```css
-.module--subcomponent {}
-/* or */
-.even-grid--item {}
+Our BEM naming doesn't need to show that we nested that image tag, so we would give it a class name of something like `.block__image`. We would _not_ give it a class name of `.block__link__image`.
+
+#### Modifier
+
+Modifiers are the most specific of the three parts of BEM. They should only define one or two properties that change the previously defined properties on the block or element. You can use a modifier on a block or an element.
+
+`.block--modifier` modifies the entire component.
+
+`.block__element--modifier` modifies a single element inside a component.
+
+#### Example
+Let's look at an example. Maybe we have a hero section that's reused on each page of a site. The block might be called `.hero`, because that's what the entire section is. The title, description, and button are all elements, so they would be `.hero__title`, `.hero__description`, and `.hero__button`. We don't need any modifiers yet, because a modifier is really specific.
+
+```html
+<section class="hero">
+  <h1 class="hero__title"></h1>
+  <p class="hero__description"></p>
+  <button class="hero__button"></p>
+</section>
 ```
 
-**Modifiers** change or add styling in certain cases. They should be separated from what they modify with *underscores*:
-```css
-.module--subcomponent_modifier {}
-/* or */
-.even-grid--item_has-image {}
+Now, what if on one page where we want a dark version of our button? We can create a modifier class for the `background-color` property so that we're only changing the color of the button with that modifier class. Notice that you still need to include the original `.hero__button` element level class. That's because the modifier is only changing the `background-color`, and you still want all the other properties from the original button class.
+
+```html
+<section class="hero">
+  <h1 class="hero__title"></h1>
+  <p class="hero__description"></p>
+  <button class="hero__button hero__button--dark"></p>
+</section>
```

Comment:
Here is where a section like this could be useful. You have plenty of example of intro BEM, and if anyone is having trouble wrapping their head around the concept, pointing them to the foundry could be a great path forward: 

`Want more examples of BEM? [Check out Nathan's Foundry article, BEM by Example][]`